ThirdParty/UIImageColors.swift

```diff
@@ -1,10 +1,31 @@
+//    UIImageColors.swift
+//    https://github.com/jathu/UIImageColors
 //
-//  UIImageColors.swift
-//  https://github.com/jathu/UIImageColors
+//    Created by Jathu Satkunarajah (@jathu) on 2015-06-11 - Toronto
+//    Original Cocoa version by Panic Inc. - Portland
 //
-//  Created by Jathu Satkunarajah (@jathu) on 2015-06-11 - Toronto
-//  Original Cocoa version by Panic Inc. - Portland
+//    MIT License
 //
+//    Copyright (c) 2015 Jathu Satkunarajah
+//
+//    Permission is hereby granted, free of charge, to any person obtaining a copy
+//    of this software and associated documentation files (the "Software"), to deal
+//    in the Software without restriction, including without limitation the rights
+//    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//    copies of the Software, and to permit persons to whom the Software is
+//    furnished to do so, subject to the following conditions:
+//
+//    The above copyright notice and this permission notice shall be included in all
+//    copies or substantial portions of the Software.
+//
+//    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//    SOFTWARE.
```

Comment:
Actually I meant adding this to our `Licenses.html` file that we show from the Settings screen. Sorry for the confusion!